src/Columns/FilterDescription.cpp

```diff
@@ -82,7 +82,11 @@ FilterDescription::FilterDescription(const IColumn & column_)
         const auto size = res.size();
         assert(size == null_map.size());
         for (size_t i = 0; i < size; ++i)
-            res[i] = res[i] && !null_map[i];
+        {
```

Comment:
Deserves a comment (basically the PR description), otherwise the next programmer reading the code is tempted to "simplify" the code to what we had before.

Reply:
OK, Robert. Thanks for the great suggestion!